src/Arcus.Testing.Tests.Integration/IntegrationTest.cs

```diff
@@ -16,7 +16,8 @@ protected IntegrationTest(ITestOutputHelper outputWriter)
             Logger = new XunitTestLogger(outputWriter);
             Configuration = TestConfig.Create(options =>
             {
-                options.AddOptionalJsonFile("appsettings.default.json");
+                options.AddOptionalJsonFile("appsettings.default.json")
+                       .AddOptionalJsonFile("appsettings.local.json");
```

Comment:
Just for you to know, since this file already had to change, used the opportunity to add this line. It makes sure that we override the `.default.json` with the `.local.json` file. It means that we can override the `DefaultAzureCredential` with our service principal credentials. Otherwise, the last file (which would be the `.default.json`) would be used.